src/DevChatter.Bot.Core/Games/Quiz/GuessQuizOperation.cs

```diff
@@ -0,0 +1,26 @@
+using DevChatter.Bot.Core.Commands.Operations;
+using DevChatter.Bot.Core.Events.Args;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace DevChatter.Bot.Core.Games.Quiz
+{
+    public class GuessQuizOperation : BaseCommandOperation
+    {
+        private readonly QuizGame _game;
+
+        public GuessQuizOperation(QuizGame game)
+        {
+            _game = game;
+        }
+
+        public override List<string> OperandWords { get; } = new List<string> { "a", "b", "c", "d" };
+
+        public override string HelpText => "Type \"!quiz a\" to guess choice \"a\" in the quiz. I think you can figure out the rest.";
+
+        public override string TryToExecute(CommandReceivedEventArgs eventArgs)
```

Comment:
I think it's misleading to name it TryXXXX since TryXXXX methods usually return a boolean. To me, this is confusing.

Reply:
It's part of the ICommandOperation interface, so it's outside the scope of this. It's a good point though. Something to consider changing at some point.